apps/docs-snippets2/scripts/wrap-snippets.ts

```diff
@@ -12,6 +12,20 @@ const wrapperFnContents = readFileSync(wrapperFnFilepath, 'utf-8');
 export const wrapSnippet = (filepath: string) => {
   const snippetContents = readFileSync(filepath, 'utf8');
 
+  /**
+   * Test environment
+   */
+  let testEnvironments = '';
+
+  // Check if the filepath includes '.node' or '.browser'
+  if (filepath.includes('.node')) {
```

Comment:
Nit but not a huge fan of the file name convention for test grouping, then we have 2 different approaches for snippets vs the rest of the repo.

Could we not just leave the `/** @group node **/` and `/** @group browser **/ in the `.ts` files? Or maybe we assume it has both envs, but if only test in a specific env if it has one of those comments?

Reply:
I see your point, I chose the latter option 5ba73a248